openmetadata/base/openmetadata-dependencies/deployments/airflow-db-migrations.yaml

```diff
@@ -70,6 +73,9 @@ spec:
         - name: db-migrations
           image: quay.io/os-climate/ingestion:0.13.1
           imagePullPolicy: IfNotPresent
+          securityContext:
```

Comment:
I wonder if we can avoid explicitly setting UID/GID allowing openshift automatically assign gid. We would need to  change scc associated with airflow service account to enable it anyways.

Reply:
Ok, I've removed the securityContext changes. These may be required, but we can merge the config changes separately and then troubleshoot if we're still seeing problems with the images not being able to properly write data.